### DISCUSSION TITLE
5eb496e no nav display

### DISCUSSION THREAD

**Discussion by SoCentral2** [flybywiresim/aircraft#4249] [Help]
I've just installed fbw 5eb496e & there's no nav display. Is it just me?

![image](https://user-images.githubusercontent.com/1389600/113481372-1ba68a00-9491-11eb-8f6e-3ec6831f2d26.png)

**Comment by ExampleWasTaken** [comment]
That is not a supported version. Download either the stable, latest development or experimental version.
